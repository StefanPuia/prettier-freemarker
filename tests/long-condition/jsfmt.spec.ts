// TODO: this should wrap
runSpec(__dirname);
