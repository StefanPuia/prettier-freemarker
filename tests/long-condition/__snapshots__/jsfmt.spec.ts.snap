// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Format: long-condition.ftl: long-condition.ftl 1`] = `
 <#if  someVar.someProp.subProp   &&	someVar && someVar1.someProp.subProp   &&	someVar1 && someVar2.someProp.subProp   &&	someVar2 && someVar3.someProp.subProp   &&	someVar3>
  <p name="foo" class="classname">Text</p>
    </#if>~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<#if someVar.someProp.subProp && someVar && someVar1.someProp.subProp && someVar1 && someVar2.someProp.subProp && someVar2 && someVar3.someProp.subProp && someVar3>
  <p name="foo" class="classname">Text</p>
</#if>

`;
